mainnet/jobs.json

```diff
@@ -1,82 +1,122 @@
 [
   {
     "weight": 1,
-    "power": 2,
+    "power": 6,
```

Comment:
hey @manuelbarbas the power for eth jobs should remain unchanged, it was only required for the skl token.